hack/install-konveyor.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Figure out where we are being run from.
+# This relies on script being run from:
+#  - ${PROJECT_ROOT}/hack/install-tackle.sh
+#  - ${PROJECT_ROOT}/bin/install-tackle.sh
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+__root="$(cd "$(dirname "${__dir}")" && pwd)"
+__repo="$(basename "${__root}")"
+__bin_dir="${__root}/bin"
+__os="$(uname -s | tr '[:upper:]' '[:lower:]')"
+__arch="$(uname -m | sed 's/x86_64/amd64/' | sed 's/aarch64/arm64/')"
+
+# Update PATH for execution of this script
+export PATH="${__bin_dir}:${PATH}"
+
+NAMESPACE="${NAMESPACE:-konveyor-tackle}"
+OPERATOR_BUNDLE_IMAGE="${OPERATOR_BUNDLE_IMAGE:-quay.io/konveyor/tackle2-operator-bundle:latest}"
+TACKLE_CR="${TACKLE_CR:-}"
+
+if ! command -v kubectl >/dev/null 2>&1; then
```

Comment:
Do you want to see if `kubectl` is on the PATH?
I think
`command kubectl 2>&-` is a little shorter and does the job as well

Reply:
The one provided doesn't handle stdout. Could make `>/dev/null` to `>-` but I think it's fine to have a few more character for clarity.

Reply:
`command kubectl` doesn't print something (only the `-v` flag does), so redirecting stdout to the null device is not necessary. Without `-v` it just changes its exit value depending on the existence of the executable on ${PATH}.

Reply:
Am I missing something?

```
bash-3.2$ command kubectl 2>&-
kubectl controls the Kubernetes cluster manager.

 Find more information at: https://kubernetes.io/docs/reference/kubectl/

Basic Commands (Beginner):
  create          Create a resource from a file or from stdin
  expose          Take a replication controller, service, deployment or pod and expose it as a new Kubernetes service
  run             Run a particular image on the cluster
  set             Set specific features on objects

...
```

Reply:
Sorry, you are completely right, my fault. You have to redirect stdout as well.

Reply:
Sorry for the confusion, I mixed up one thing with the `which` command: `command` without the `-v` flag will even execute the given executable. Thats the reason why you see the standard output, it is just the help text of `kubectl` - be _cautious_: other commands may even have side effects. Hence, you should definitely use `command -v ...` as in the beginning, as it will only check the existence of the executable (and print its path).

Reply:
I was verifying only with `command false` and `command true` which doesn't print something at all (but nevertheless may have side effects, depending on the given exe).